homedecor_seating/init.lua

```diff
@@ -72,8 +72,8 @@ end
 function lrfurn.fix_sofa_rotation_nsew(pos, placer, itemstack, pointed_thing)
 	local node = minetest.get_node(pos)
 	local colorbits = node.param2 - (node.param2 % 8)
-	local yaw = placer:get_look_yaw()
-	local dir = minetest.yaw_to_dir(yaw-1.5)
```

Comment:
What was the -1.5 for?

Reply:
`-1.5` was wrong, it should have been `-1.5708`.
`get_look_yaw()` returns the +x direction and `get_look_horizontal()` returns the +z direction. The -1.5 was a workaround to get the right direction.